test/unit/src/common/actions/t_snap-builds.js

```diff
@@ -1,297 +1,114 @@
 import expect from 'expect';
-import configureMockStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
-import nock from 'nock';
-import { isFSA } from 'flux-standard-action';
-
-import { conf } from '../../../../../src/server/helpers/config';
 
 import {
   fetchSnap,
   fetchBuilds,
-  requestBuilds,
-  fetchBuildsSuccess,
-  fetchBuildsError
+  requestBuilds
 } from '../../../../../src/common/actions/snap-builds';
 import * as ActionTypes from '../../../../../src/common/actions/snap-builds';
-
-const middlewares = [ thunk ];
-const mockStore = configureMockStore(middlewares);
+import { CALL_API } from '../../../../../src/common/middleware/call-api';
 
 describe('snap builds actions', () => {
-  let store;
   let action;
 
   const repo = 'dummy/repo';
   const repositoryUrl = `https://github.com/${repo}`;
 
-  beforeEach(() => {
-    store = mockStore({});
-  });
-
-  context('fetchBuildsSuccess', () => {
-    let builds = [ { build: 'test1' }, { build: 'test2' }];
+  context('fetchBuilds', () => {
+    const snapUrl = 'https://api.launchpad.net/devel/~foo/+snap/bar';
```

Comment:
No - the action creators are tested as units without the mock dispatcher. This is possible because action creators like `fetchBuilds` only return declarative data - they don't call the fetch api as a side-effect.